src/components/widgets-settings/width/width.directive.coffee

```diff
@@ -62,14 +62,13 @@ module.controller('SettingWidthCtrl', ($scope, $element, $timeout, $log, ImpacWi
   $scope.pdfMode = false
   ImpacDashboardsSvc.pdfModeEnabled().then(null, null, ->
     $scope.pdfMode = true
-    $scope.initiallyExpanded = $scope.expanded
     # Expand the widget if it's not already the case
     w.toggleExpanded(false) unless $scope.expanded
   )
   ImpacDashboardsSvc.pdfModeCanceled().then(null, null, ->
     $scope.pdfMode = false
     # Reduce the widget if it wasn't expanded initially
-    w.toggleExpanded(false) unless $scope.initiallyExpanded
+    w.toggleExpanded(false) unless w.initiallyExpanded
```

Comment:
my mistake: it's not initialized in DashboardSvc, but in widget.directive. I'll describe in another comment